src/olympia/lib/settings_base.py

```diff
@@ -1060,10 +1060,21 @@ def get_db_config(environ_var, atomic_requests=True):
 # a separate, shorter timeout for validation tasks.
 CELERY_TASK_SOFT_TIME_LIMIT = 60 * 30
 
+# List of modules that contain tasks and that wouldn't necessarily be imported
+# automatically when celery starts.
 CELERY_IMPORTS = (
+    'olympia.activity.tasks',
+    'olympia.bandwagon.tasks',
+    'olympia.files.tasks',
     'olympia.lib.crypto.tasks',
     'olympia.lib.es.management.commands.reindex',
+    'olympia.ratings.tasks',
+    'olympia.reviewers.tasks',
+    'olympia.versions.tasks',
     'olympia.stats.management.commands.index_stats',
+    'olympia.tags.tasks',
+    'olympia.users.tasks',
+    'olympia.zadmin.tasks',
```

Comment:
This is missing `celery.scanners.tasks`

Reply:
You mean `olympia.scanners.tasks` ? It's not needed because it ends up automatically imported anyway. This setting is not an exhaustive list, it's just everything that doesn't get automatically imported at startup (and a few more that are automatically started at startup but not in tests as well)